backend/tests/test_profile.py

```
@@ -0,0 +1,167 @@
+            "username": "testuser",
+            "password": "testpass"
+        }
+    )
+    assert response.status_code == 200, "Failed to obtain token"
+    token = response.json()["access_token"]
+    return {"Authorization": f"Bearer {token}"}
+
+# ----------------------
+# ✅ Test complete_profile_step1 success
+# ----------------------
+def test_complete_profile_step1_success(auth_headers):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_complete_profile_step1_success`

Add return type annotation: `None`

(ANN201)